docs/spec/lightclient/verification/Lightclient_A_1.tla

```diff
@@ -115,8 +135,8 @@ ValidAndVerified(trusted, untrusted) ==
     THEN "FAILED_TRUSTING_PERIOD" 
     ELSE IF untrusted.header.height = trusted.header.height + 1
              \/ SignedByOneThirdOfTrusted(trusted, untrusted)
-         THEN "OK"
-         ELSE "CANNOT_VERIFY"
+         THEN "SUCCESS"
+         ELSE "NOT_ENOUGH_TRUST"
```

Comment:
I think OK and CANNOT_VERIFY are used in the English spec. Are SUCCESS and NOT_ENOUGH_TRUST used in the implementation? If yes, we should adapt I guess the English spec.

Reply:
I've taken the verdicts from the implementation. There is another issue I've got, namely that according to the TLA+ constraints the verdict FAILED_TRUSTING_PERIOD cannot ever be produced. At least it's impossible to obtain the a test with such verdict from Apalache, and my quick manual check on the TLA+ constraints tells me it's indeed impossible. Could you please verify that? In case it's true, it may be makes sense to remove such a verdict. And I would actually update the english spec as well, as I find the implementation names a bit more informative

Reply:
I will have a look with @konnov

Reply:
I don't see why `FAILED_TRUSTING_PERIOD` cannot be produced. It is produced in line 115. However, it is not propagated into `state`. Do you like to see it in `state`?

Reply:
this PR is closed; let's discuss on another PR: [546](https://github.com/informalsystems/tendermint-rs/pull/546)